### PR TITLE
Setup jest in checkout

### PR DESCRIPTION
**Pull request by bmigirl**
Also remove some code that actually wasn't is use anymore

### CODE REVIEW

File: packages/checkout-storefront/src/sections/Addresses/utils.ts [hunk 1]
```diff
@@ -92,3 +93,41 @@ export const getAddressVlidationRulesVariables = (
...
+  "streetAddress1",
+  "streetAddress2",
+  "city",
+  "postalCode",
+  "cityArea",
+  "countryArea",
+];
+
+// api doesn't order the fields but we want to
+export const getOrderedAddressFields = (addressFields: AddressField[] = []): AddressField[] => {
```
Comment (bmigirl) [inline packages/checkout-storefront/src/sections/Addresses/utils.ts:111]: Threw in something extra for you @mmiszy 😃

Comment (mmiszy) [reply packages/checkout-storefront/src/sections/Addresses/utils.ts:111]: 😆

**Review by mmiszy — APPROVED** [review]
Well done, looking good

File: packages/checkout-storefront/package.json [hunk 2]
```diff
@@ -11,7 +11,9 @@
     "generate": "DOTENV_CONFIG_PATH=.env graphql-codegen -r env-vars --config codegen.yml",
     "check-types": "tsc --noEmit",
     "lint": "eslint --max-warnings=0 \"src/**/*.@(tsx|ts|jsx|js)\" --fix",
-    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
+    "test": "jest",
+    "test-dev": "jest --watch"
```
Comment (witoszekdev) [inline packages/checkout-storefront/package.json:16]: Can we use that instead? I think we have this pattern in main `package.json`, so might as well use it here as well :P
```suggestion
    "test:dev": "jest --watch"
```

**Review by witoszekdev — COMMENTED** [review]
really nice 🥳

File: packages/checkout-storefront/package.json (hunk 2 above)
Comment (bmigirl) [reply packages/checkout-storefront/package.json:16]: Okay

**Review by mmiszy — APPROVED** [review]

**Pull request merged**